lib/action_view/component/railtie.rb

```diff
@@ -35,9 +35,7 @@ class Railtie < Rails::Railtie # :nodoc:
 
       initializer "action_view_component.eager_load_actions" do
         ActiveSupport.on_load(:after_initialize) do
-          ActionView::Component::Base.descendants.each do |descendant|
-            descendant.compile if descendant.has_initializer? && config.eager_load
-          end
+          ActionView::Component::Base.descendants.each(&:compile)
```

Comment:
Does this mean we'll always be compiling components even if eager loading is turned off?

Reply:
I don't think so. I think i tested turning eager load off in test.rb and the spec fails.

Reply:
Confirmed locally ☝️